testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -317,7 +317,6 @@ def update_props():
 @pytest.mark.xattrs
 @pytest.mark.session
 @pytest.mark.parametrize('sg_conf_name, x509_cert_auth', [
-    ('xattrs/no_import', True),
     ('xattrs/no_import', False)
```

Comment:
same question regarding to x509 flag removal. just to confirm x509 is not necessary here to test.